Description: Fix deprecated trigger_error.
Author: Rinat Gumirov <rinat.gumirov@mail.ru>
Last-Update: 2025-11-28

--- a/pts-core/objects/pts_openbenchmarking.php
+++ b/pts-core/objects/pts_openbenchmarking.php
@@ -177,7 +177,7 @@
 		}
 		else if(PTS_IS_CLIENT && isset($json_response['openbenchmarking']['result']['error']))
 		{
-			trigger_error($json_response['openbenchmarking']['result']['error'], E_USER_ERROR);
+			exit($json_response['openbenchmarking']['result']['error']);
 		}
 
 		return $valid;
@@ -650,7 +650,7 @@
 			{
 				if(!defined('PHOROMATIC_SERVER'))
 				{
-					trigger_error('Unable to obtain ' . $qualified_identifier . ' from OpenBenchmarking.org. ' . (!pts_network::internet_support_available() ? 'Internet connection disabled/unavailable.' : 'If this issue persists, file an issue @ https://github.com/phoronix-test-suite/phoronix-test-suite/issues') . PHP_EOL, E_USER_ERROR);
+					exit('Unable to obtain ' . $qualified_identifier . ' from OpenBenchmarking.org. ' . (!pts_network::internet_support_available() ? 'Internet connection disabled/unavailable.' : 'If this issue persists, file an issue @ https://github.com/phoronix-test-suite/phoronix-test-suite/issues') . PHP_EOL);
 				}
 				return false;
 			}
@@ -957,7 +957,7 @@
 			{
 				if(!defined('PHOROMATIC_SERVER'))
 				{
-					trigger_error('Unable to obtain ' . $qualified_identifier . ' from OpenBenchmarking.org. ' . (!pts_network::internet_support_available() ? 'Internet connection disabled/unavailable.' : 'If this issue persists, file an issue @ https://github.com/phoronix-test-suite/phoronix-test-suite/issues') . PHP_EOL, E_USER_ERROR);
+					exit('Unable to obtain ' . $qualified_identifier . ' from OpenBenchmarking.org. ' . (!pts_network::internet_support_available() ? 'Internet connection disabled/unavailable.' : 'If this issue persists, file an issue @ https://github.com/phoronix-test-suite/phoronix-test-suite/issues') . PHP_EOL);
 				}
 				return false;
 			}
